src/web/handlers/GestureHandler.ts

```diff
@@ -352,6 +405,65 @@ export default abstract class GestureHandler {
     };
   }
 
+  private transformTouchEvent(
+    event: AdaptedEvent
+  ): ResultTouchEvent | undefined {
+    if (!event.allTouches || !event.changedTouches || !event.touchEventType) {
+      return;
+    }
+
+    const rect = this.view.getBoundingClientRect();
+
+    const all: PointerData[] = [];
+    const changed: PointerData[] = [];
+
+    // eslint-disable-next-line @typescript-eslint/prefer-for-of
+    for (let i = 0; i < event.allTouches.length; ++i) {
+      const id: number = this.tracker.getMappedTouchEventId(
+        event.allTouches[i].identifier
+      );
+
+      if (isNaN(id)) return;
```

Comment:
This shouldn't be, but it does because of TouchEvents structure and the way we handle them right now. I agree that it doesn't look good and I'll take a look at it to find better way to solve this issue, but for now it has to stay. I'll add comment with explanation